src/utils/Fields/renderOptGroup.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function renderOptGroup( opt, optStyles ) {
+  return (
+    <optgroup label={ opt.group }>
```

Comment:
there's a weird thing where on initial load, this optgroup label is checked - it's not a blocker for merging this PR but we should fix eventually